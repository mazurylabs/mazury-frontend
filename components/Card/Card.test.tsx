import { render, screen } from '@testing-library/react';
import { Card } from '.';

test('Card', () => {
  render(<Card>Card content</Card>);
  const cardContainer = screen.getByText('Card content');
  expect(cardContainer).toBeInTheDocument();
  expect(cardContainer).toMatchSnapshot();
});
