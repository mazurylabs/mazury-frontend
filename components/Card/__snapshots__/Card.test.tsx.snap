// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card 1`] = `
<div
  class="rounded-2xl border border-indigoGray-20 p-6 transition-all duration-100 undefined cursor-pointer bg-white shadow-base hover:shadow-lg active:bg-indigoGray-5 active:shadow-none"
>
  Card content
</div>
`;
